web-common/src/features/dashboards/time-series/MetricsTimeSeriesCharts.svelte

```diff
@@ -200,7 +200,7 @@
   start={startValue}
   {workspaceWidth}
 >
-  <div class="bg-white sticky top-0 flex pl-1" style="z-index:100">
+  <div class="bg-white sticky top-0 flex pl-1">
```

Comment:
Scrolling the charts container is buggy. The top axis should be always clearly visible when scrolling the container.
![image](https://github.com/rilldata/rill/assets/4402679/fd6eb226-98bd-4f40-a406-89d4baf3e23b)